hack/jenkins/e2e.sh

```diff
@@ -217,6 +217,9 @@ if [[ "${E2E_UP,,}" == "true" ]]; then
         # other.
         export KUBE_SKIP_UPDATE=y
         sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update preview -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update alpha -q" || true
```

Comment:
I'm confused. You don't need any of these. `update` updates everything - I've watched it. This code should be redundant - the code in `util.sh` has been redundant for a long time, and I've been meaning to clean it up.

The `update alpha -q` / etc. is only useful for guaranteeing the component is installed, which we don't need here.

cc @jlowdermilk